### PR TITLE
Updates based on review from ckittel - 1/2

### PR DESCRIPTION
**Pull request by norelina**
## Purpose

Updates based on review from @ckittel - 1/2

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Review by olasowemimo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -42,14 +42,29 @@ The flow below demonstrates how a user can access a service that is running on a
...
+        - Microsoft.Web
+        - Microsoft.Storage
 - [Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli) 2.4.2 or newer
 - [Azure CLI IoT extension](https://github.com/Azure/azure-iot-cli-extension#installation)
 - [Powershell](https://learn.microsoft.com/powershell/scripting/install/installing-powershell)
 - [.NET 6.0](https://dotnet.microsoft.com/download/dotnet/6.0)
 
 ### Provision Azure components
 
+1. Clone the repo
```
Comment (ckittel) [inline README.md]:
```suggestion
1. Clone the repo.
```

File: README.md [hunk 2]
```diff
@@ -42,14 +42,29 @@ The flow below demonstrates how a user can access a service that is running on a
...
+
+    ```
```
Comment (ckittel) [inline README.md]:
~~~suggestion
    ```bash
~~~

File: README.md [hunk 3]
```diff
@@ -42,14 +42,29 @@ The flow below demonstrates how a user can access a service that is running on a
 
 ### Prerequisites
 
-- [Docker](https://www.docker.com/)
+- [Azure Subscription](https://azure.microsoft.com/) with
+    - permissions to create resources and perform role assignments
+    - the following [Resource providers](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types#register-resource-provider) registered
```
Comment (ckittel) [inline README.md]:
```suggestion
    - the following [Resource providers](https://learn.microsoft.com/azure/azure-resource-manager/management/resource-providers-and-types#register-resource-provider) registered
```

File: README.md [hunk 4]
```diff
@@ -93,7 +108,11 @@ The flow below demonstrates how a user can access a service that is running on a
 
 To start the simulated device follow these steps:
 
-1. Confirm that during the provisioning step, the **src/simulated-device/app.config** was generated.
+1. Review the **src/simulated-device/app.config** to confirm that the values for the following appSettings were set during the provisioning step.
+    - IOTHUB_DEVICE_CONNECTION_STRING
+    - AZRELAY_CONN_STRING
+    - SERVICE_PROTOCOL
+    - SERVICE_PORT
```
Comment (ckittel) [inline README.md]:
```suggestion
    - `IOTHUB_DEVICE_CONNECTION_STRING`
    - `AZRELAY_CONN_STRING`
    - `SERVICE_PROTOCOL`
    - `SERVICE_PORT`
```

File: deploy/provision_resources.ps1 [hunk 5]
```diff
@@ -96,24 +95,17 @@ Write-Host "Creating simulated-device app.config..."
...
-    docker push $LocalForwarderImage
-
-    Write-Host "Building the docker image for the azure function..."
-    docker build ../src/function/. -f ../src/function/Dockerfile --tag $FunctionImage
-
-    Write-Host "Pushing the azure function image to the Azure Container Registry..."
-    docker push $FunctionImage
+    Write-Host "Building and pushing the container image for the local forwarder..."
+    az acr build --registry $Azure.ContainerRegistry.Name --image $LocalForwarderImage --platform linux/amd64 ../src/local-forwarder/
+    Write-Host "Building and pushing the container image for the Azure Function..."
```
Comment (ckittel) [inline deploy/provision_resources.ps1:101]:
```suggestion

    Write-Host "Building and pushing the container image for the Azure Function..."
```

**Review by ckittel — APPROVED** [review]
Awesome!  :shipit:

**Pull request merged**